app/components/markdown/markdown_link/index.js

```diff
@@ -5,14 +5,18 @@ import {bindActionCreators} from 'redux';
 import {connect} from 'react-redux';
 
 import {getConfig, getCurrentUrl} from '@mm-redux/selectors/entities/general';
+import {getCurrentTeam} from '@mm-redux/selectors/entities/teams';
 import {handleSelectChannelByName} from 'app/actions/views/channel';
 
 import MarkdownLink from './markdown_link';
 
 function mapStateToProps(state) {
+    const team = getCurrentTeam(state);
+
     return {
         serverURL: getCurrentUrl(state),
         siteURL: getConfig(state).SiteURL,
+        currentTeamName: team && team.name,
```

Comment:
I think this is also equivalent 

`currentTeamName: getCurrentTeam(state)?.name,`